python/scylla/_rust/execution_profile.pyi

```diff
@@ -0,0 +1,12 @@
+from .enums import Consistency, SerialConsistency
+
+class ExecutionProfile:
+    def __init__(
+        self,
+        timeout: float | None = None,
+        consistency: Consistency | None = None,
+        serial_consistency: SerialConsistency | None = None,
+    ) -> None: ...
+    def get_request_timeout(self) -> float | None: ...
+    def get_consistency(self) -> Consistency: ...
+    def get_serial_consistency(self) -> SerialConsistency | None: ...
```

Comment:
@wprzytula Please review this part carefully. I see that `get_consistency` return non-optional consistency, but `get_serial_consistency` returns optional serial consistency. I remember there were some reasons for similar weirdness in Rust Driver, but I don't remember enough to correctly asses this code.